pkg/client/status_reporter.go

```diff
@@ -43,9 +43,8 @@ func NewStatusReporter(client clientv1.ClusterOperatorInterface, name, namespace
 
 func newRelatedObjects(namespace string) []v1.ObjectReference {
 	return []v1.ObjectReference{
-		{Resource: "namespaces", Name: namespace},
 		// Gather pods, services, daemonsets, deployments, replocasets, statefulsets, and routes
-		{Resource: "all", Name: namespace},
```

Comment:
We then need to adjust the comment above this?

Reply:
Never. This is downloading what is said in a comment.
`{Resource: "namespaces", Name: namespace}` will download all resources from `namespace`, which in our case is `openshift-monitoring`.

Reply:
Okay, I see now, I found the comment confusing, but if everyone else thinks it clear. :shipit: